tests/core/core.spec.ts

```diff
@@ -310,9 +310,9 @@ suite("@dataform/core", () => {
         schema: "schema"
       });
       const graph = session.compile();
-      expect(graph.graphErrors.compilationErrors.map(error => error.message)).deep.equals([
+      expect(graph.graphErrors.compilationErrors.map(error => error.message)).deep.equals(Array(2).fill(
```

Comment:
[check uniqueness](https://github.com/dataform-co/dataform/blob/e42c57bce81221bc36e92a66c0bf107244b63ddf/core/session.ts#L597) methods haven't been including all duplicate actions. Because the first duplicate action wasn't detecting.